ddtrace/tracer/util.go

```diff
@@ -35,6 +40,9 @@ func toFloat64(value interface{}) (f float64, ok bool) {
 	case uint32:
 		return float64(i), true
 	case uint64:
+		if i > uint64(max) {
```

Comment:
Why is type casting to `uint64(max)` necessary here? Cannot be avoided?

Reply:
It's necessary. We can't compare uint64 with int64 without a cast.

Reply:
good to know 😄